### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by Feichtmeier**
Hello there :D

Really impressed with your project!

If wanted, here is a pull request with some styling changes. If not you can close it :) Best regards,

Changes are:

- add and use yaru and yaru_icons
- remove the pink accent colors
- add padding to the download options list

grafik
grafik
grafik
grafik
grafik
grafik

### CODE REVIEW

File: lib/src/app.dart [hunk 1]
```diff
@@ -12,9 +13,9 @@ class _AppState extends State<App> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      theme: ThemeData(primarySwatch: Colors.pink),
-      darkTheme: ThemeData.dark(),
-      home: const MainPage(title: 'Quickgui - A Flutter frontend for Quickget and Quickemu'),
-    );
+        theme: yaru.lightTheme,
```
Comment (diddledani) [inline lib/src/app.dart:16]: I think we wanted to retain the pink accent colours, though I might be mistaken?

Comment (ymauray) [reply lib/src/app.dart:16]: Nop, you are absolutely right ! The pink is here to stay ! :-)

**Comment by ymauray** [comment]
Thanks for your pull request. I will have a look at it, but I can tell you right now that the pink accent is not going anywhere :-)

There are interesting ideas in your contribution, and I'll probably integrate some of them and credit you properly.

I'll keep the PR opened until I'm done !

**Comment by Feichtmeier** [comment]
> Thanks for your pull request. I will have a look at it, but I can tell you right now that the pink accent is not going anywhere :-)
> 
> There are interesting ideas in your contribution, and I'll probably integrate some of them and credit you properly.
> 
> I'll keep the PR opened until I'm done !

Alright! Branding is ofc important :)

**Pull request closed without merging**